projects/core/directives/dropdown/dropdown-position-sided.directive.ts

```diff
@@ -10,13 +10,12 @@ import {TuiDropdownPosition} from './dropdown-position.directive';
 @Directive({
     standalone: true,
     selector: '[tuiDropdownSided]',
-    providers: [TuiDropdownPosition, tuiAsPositionAccessor(TuiDropdownPositionSided)],
+    providers: [tuiAsPositionAccessor(TuiDropdownPositionSided)],
```

Comment:
Looks like this was still necessary. Please put it back — I'll investigate myself later how so.